docs/user-guide/expressions/functions.md

```diff
@@ -11,25 +11,82 @@ In the examples below we will use the following `DataFrame`:
 --8<-- "python/user-guide/expressions/functions.py:dataframe"
 ```
 
-
-#### Column Selection
+## Column Selection
```

Comment:
I think that this should be in its own file/page.

I think we should start first with explaining `expression expansion` with a few examples. And once that concept is clear really show of with the selector API.

This will make clear that the `selector` API is actually expression expansion and thus can really make code very DRY.